cloud/deploys/dev_azure/civiform_config.example.sh

```diff
@@ -0,0 +1,34 @@
+export CIVIFORM_CLOUD_PROVIDER="azure"
+export CIVIFORM_MODE="dev"
+export CIVIC_ENTITY_SHORT_NAME="Sooschester"
+export CIVIC_ENTITY_FULL_NAME="City of Sooschester"
+export CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="CHANGE ME"
+export CIVIC_ENTITY_LOGO_WITH_NAME_URL="https://www.cityofrochester.gov/images/header-logo.png"
+export CIVIC_ENTITY_SMALL_LOGO_URL="https://www.cityofrochester.gov/assets/0/117/8589934986/c6791498-270e-40b1-9b81-aca8a510c1b5.jpg"
+export EMAIL_SENDER="ses"
+export ADMIN_AUTH_PROVIDER="activeDirectory"
+export ADMIN_AUTH_FORMAT="OIDC"
+export RESIDENT_AUTH_PROVIDER="loginRadius"
+export APPLICATION_NAME="CHANGE ME"
+export DOCKER_REPOSITORY_NAME="civiform"
+export DOCKER_USERNAME="civiform"
+export IMAGE_TAG="latest"
+export SENDER_EMAIL_ADDRESS="CHANGE ME"
+export STAGING_HOSTNAME=""
+export STAGING_PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export STAGING_TI_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export STAGING_APPLICANT_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export TERRAFORM_TEMPLATE_DIR="cloud/azure/templates/azure_saml_ses"
+export AZURE_LOCATION="eastus"
+export AZURE_LOG_STORAGE_ACCOUNT_NAME="CHANGE ME"
+export AZURE_SUBSCRIPTION="ec814e56-5241-4d0b-9cd2-02d608c99732"
```

Comment:
nit: is this a real value in here?

Reply:
https://docs.microsoft.com/en-us/answers/questions/340725/is-subscription-id-personal-or-should-be-kept-priv.html#:~:text=In%20your%20case%20it%20is,are%20the%20users%20%2F%20owners%20etc.

Reply:
I was thinking this should be another "CHANGE ME" to make it clear that you need to replace this value.